packages/ui5-application-writer/test/data.test.ts

```diff
@@ -113,6 +113,42 @@ describe('Setting defaults', () => {
                 ui5Theme: 'sap_fiori_3',
                 ui5Libs: defaultUI5Libs
             }
+        },
+        {
+            input: {
+                version: 'snapshot-1.80',
+                frameworkUrl: 'https://ui5.unreleased.sap.com'
+            },
+            expected: {
+                framework: 'SAPUI5',
+                frameworkUrl: 'https://ui5.unreleased.sap.com',
+                version: 'snapshot-1.80',
+                localVersion: 'snapshot-1.80',
```

Comment:
Local version should be `1.80.0`

Reply:
@devinea @IainSAP What was the `localVersion` again? The version for the `ui5-local.yaml`? If yes, then `localVersion` can't be a snapshot. Only released UI5 versions 1.76.0 and above.

Reply:
@zdravko-georgiev  Yeah `localVersion` is the one for `ui5-local.yaml`
@IainSAP  - Okay yeah I guess if `localVersion` is passed in as `snapshot-1.80` it should be output the same. Even if it is not really valid for the ui5-local.yaml.